x/fundraising/spec/01_concepts.md

```diff
@@ -4,18 +4,71 @@
 
 ## Fundraising Module
 
-The `x/fundraising` Cosmos SDK module provides an oppotunity for new projects to onboard the ecosystem. It does not only allow projects to raise funds, but also increase their brand awareness before launching their projects.
+The `x/fundraising` Cosmos SDK module is a module to raise funds as an auction of tokens. This fundraising module provides an opportunity for a new project to onboard the ecosystem. It does not only allow the project to raise funds, but also increase its brand awareness before launching the project.
 
 ## Auction Types
 
-There are two different types of auctions in the fundraising module.
+This fundraising module provides two different types of auctions: 1) Fixed Price Auction and 2) Batch Auction.
 
-### Fixed Price Auction
+## Fixed Price Auction
 
-A fixed price auction type is the most basic type of an auction that is first come first served way to raise funds. When an autioneer creates this fixed price auction, they must determine the fixed start price that is proportional to the paying coin denom. Once it is created, bidders can only bid with the start price of the auction. For example, an auctioneer sets 0.5 as start price for X coin and the paying coin is Y coin. How many X coins does a bidder receives if they bids 10Y coin? It is calculated as Y coin over start price (10/0.5), which results to 20X coin.  
+This fixed price auction is to sell a given amount of tokens on a first-come, first-served basis.
 
-### English Auction
+### What an auctioneer does:
+When an auctioneer creates a fixed price auction, it must determine the following parameters.
 
-An english auction type is an ascending dynamic auction that an auctioneer decides the starting price of the selling amount of coin and bidders bid to purchase the amounts of the coin. Competition for the price, not how fast bidders can bid.
+- **Selling Token**: the denom of tokens to be auctioned,
+- **Paying Token**: the denom of tokens to be used for payment,
+- **Price**: fixed amount of the paying tokens to get a selling token (i.e., amount of paying tokens per selling token),
+- **Auction Start Time**: when the auction starts,
+- **Auction End Time**: when the auction ends,
+- **Offering Quantity**: total amount of selling tokens to be auctioned.
```

Comment:
I update names to be the exact names that are used in the codebase and add `Vesting Schedules`. You can reference what parameters that an auctioneer needs to input in [this /docs/How-To/cli/README.md](https://github.com/tendermint/fundraising/tree/main/docs/How-To/cli#createfixedpriceauction).
```suggestion
- **Price**: fixed amount of the paying tokens to get a selling token (i.e., amount of paying tokens per selling token),
- **Selling Coin**: the total amount of selling coin to be auctioned,
- **Paying Coin Denom**: the denom of coin to be used for payment,
- **Vesting Schedules**: the vesting schedules to receive the sold amounts,
- **Auction Start Time**: when the auction starts,
- **Auction End Time**: when the auction ends.
```

Reply:
Good suggestions. I made one more change to add "denom and" to Selling Coin.
```suggestion
- **Price**: fixed amount of the paying tokens to get a selling token (i.e., amount of paying tokens per selling token),
- **Selling Coin**: the denom and total amount of selling coin to be auctioned,
- **Paying Coin Denom**: the denom of coin to be used for payment,
- **Vesting Schedules**: the vesting schedules to receive the sold amounts,
- **Auction Start Time**: when the auction starts,
- **Auction End Time**: when the auction ends.
```

We are now using both "token" and "coin" here and there. What do you think we use one term between two? Which one is better you think?

Reply:
We have internally discussed this before. We decided to use `coin` because both selling coin and paying coin are native assets that operate on its own blockchain.

Reply:
Thanks for the info. I will modify accordingly.